lib/middlewares/settingsCache.js

```diff
@@ -0,0 +1,26 @@
+const _ = require('lodash/fp')
+const state = require('./state')
+
+const getTimestamp = (operatorId) => state.settingsCache[operatorId] ? state.settingsCache[operatorId].timestamp : null
+
+const getCache = (operatorId) => state.settingsCache[operatorId] ? state.settingsCache[operatorId].cache : null
```

Comment:
Might be a good idea to check for null or blank operatorId. Otherwise, if it's not passed in properly we could cross-pollute operator info.